mne/io/eyelink/eyelink.py

```diff
@@ -210,8 +210,8 @@ def __init__(
         self.fname = Path(fname)
         self._sample_lines = None  # sample lines from file
         self._event_lines = None  # event messages from file
+        self._raw_extras = list([dict()])  # extra info from file
```

Comment:
```suggestion
        raw_extras = dict()  # extra info from file
```

Reply:
Thanks for the tips! I can make raw_extras a variable as you suggest here, but from what I can tell I'll just need to define it outside of the `RawEyelink` class so that it is accessible to the various methods/funcs that access it before the `Super` call.